contracts/seed/Seed.sol

```diff
@@ -127,132 +135,157 @@ contract Seed {
       * @param _price                 The price in wei of fundingTokens when exchanged for seedTokens.
       * @param _startTime             Distribution start time in unix timecode.
       * @param _endTime               Distribution end time in unix timecode.
-      * @param _vestingDuration       Vesting period duration in days.
-      * @param _vestingCliff          Cliff duration in days.
-      * @param _permissionedSeed         Set to true if only whitelisted adresses are allowed to participate.
+      * @param _vestingDuration       Vesting period duration in seconds.
+      * @param _vestingCliff          Cliff duration in seconds.
+      * @param _permissionedSeed      Set to true if only whitelisted adresses are allowed to participate.
       * @param _fee                   Success fee expressed as a % (e.g. 2 = 2% fee)
     */
     function initialize(
         address _beneficiary,
         address _admin,
         address[] memory _tokens,
-        uint256[] memory _softAndHardCap,
+        uint256[] memory _softHardThresholds,
         uint256 _price,
         uint256 _startTime,
         uint256 _endTime,
-        uint16  _vestingDuration,
-        uint16  _vestingCliff,
+        uint32  _vestingDuration,
+        uint32  _vestingCliff,
         bool    _permissionedSeed,
         uint8   _fee
     ) public initializer {
-        beneficiary     = _beneficiary;
-        admin           = _admin;
-        softCap         = _softAndHardCap[0];
-        hardCap         = _softAndHardCap[1];
-        price           = _price;
-        startTime       = _startTime;
-        endTime         = _endTime;
-        vestingDuration = _vestingDuration;
-        vestingCliff    = _vestingCliff;
-        permissionedSeed   = _permissionedSeed;
-        seedToken       = IERC20(_tokens[0]);
-        fundingToken    = IERC20(_tokens[1]);
-        fee             = _fee;
-        closed          = false;
-        minimumReached  = false;
-        maximumReached  = false;
-        seedRemainder   = IERC20(_tokens[0]).balanceOf(address(this));
+        beneficiary       = _beneficiary;
+        admin             = _admin;
+        softCap           = _softHardThresholds[0];
+        hardCap           = _softHardThresholds[1];
+        price             = _price;
+        startTime         = _startTime;
+        endTime           = _endTime;
+        vestingDuration   = _vestingDuration;
+        vestingCliff      = _vestingCliff;
+        permissionedSeed  = _permissionedSeed;
+        seedToken         = IERC20(_tokens[0]);
+        fundingToken      = IERC20(_tokens[1]);
+        fee               = _fee;
+        closed            = false;
+        minimumReached    = false;
+        maximumReached    = false;
+
+        // Same calculation as we did at SeedFactory.deploySeed()
+        uint256 seedAmountSent = (_softHardThresholds[1].div(_price)).mul(10**18);
```

Comment:
The amount may not have been sent at this point, so this variable name is misleading, confusing.  Can we just set seedRemainder directly?

use hardCap instead of _softHardThresholds[1].

Why do we have to be doing these wei conversions?  Everything here is in wei, no?

Reply:
In this PR, the SeedFactory sends the tokens, but have made changes, it will be in the new PR.